samples/count_tokens.py

```diff
@@ -24,16 +24,14 @@ class UnitTests(absltest.TestCase):
     def test_tokens_context_window(self):
         # [START tokens_context_window]
         model_info = genai.get_model("models/gemini-1.0-pro-001")
-        # Returns the "context window" for the model (the combined input and output token limits)
+
+        # Returns the "context window" for the model,
+        # which is the combined input and output token limits.
         print(f"{model_info.input_token_limit=}")
```

Comment:
Here the output is printed as `key=value`, whereas in most other places it's `key: value`.
Is there some system/method behind this?